package.json

```diff
@@ -23,7 +23,7 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.14",
     "@fortawesome/free-solid-svg-icons": "^5.7.1",
     "@fortawesome/react-fontawesome": "^0.1.4",
-    "@material-ui/core": "^3.9.2",
+    "@material-ui/core": "^4.1.3",
```

Comment:
`4.2.1` has been released, might as well update this now.

Reply:
Updated